gammapy/utils/fitting/fit.py

```diff
@@ -20,53 +22,152 @@ class FitMeta(InheritDocstrings, abc.ABCMeta):
 class Fit(object):
     """Abstract Fit base class.
     """
-
     @abc.abstractmethod
     def total_stat(self, parameters):
         """Total likelihood given the current model parameters"""
         pass
 
-    def fit(self, opts_minuit=None):
-        """Run the fit
+    def optimize(self, optimizer="minuit", opts_minuit=None):
```

Comment:
Rename this argument `opts_minuit` to `opts`, and then just pass it to the Sherpa interface as well?
We could introduce a uniform optimizer config and translation to the backends, or just pass things along without modification.
either way, having a single optional `opts` argument here would be better than having backend-specific arguments, no?